TikTokTrainer/TikTokTrainerApp.swift

```diff
@@ -14,4 +15,17 @@ struct TikTokTrainerApp: App {
             ContentView()
         }
     }
+    /// Runs the poseNetProcessor on the uploaded videos
+    /// Runs the two different run functions concurrently with promises
+    init() {
+        all(
+            PoseNetProcessor.run(url: Bundle.main.url(forResource: "preRecordedVid", withExtension: "mp4")!),
+            PoseNetProcessor.run(url: Bundle.main.url(forResource: "recordedVid", withExtension: "mp4")!)
+        ).then { movieOne, movieTwo in
+            let error = try ScoringFunction(preRecordedVid: movieOne, recordedVid: movieTwo).computeUnweightedMSE()
+            print(error)
+        }.catch { error in
+            print("Error: \(error)")
+        }
+    }
```

Comment:
Can we get rid of this before it gets merged into develop? You could link this call up to the uploaded/user-recorded videos.